packages/xo-server/config.toml

```diff
@@ -75,6 +75,7 @@ defaultSignInPage = '/signin'
 # This is used to mitigate bruteforce attacks without being visible to users.
 throttlingDelay = '2 seconds'
 
+# See https://github.com/vatesfr/xen-orchestra/blob/master/%40xen-orchestra/backups/README.md#changing-the-default-backup-configuration
```

Comment:
No documentation exists to explain autoUnmountPartitionDelay and disableMergeWorker? (just below)

Reply:
I don't think adding a comment to explain each value is necessary, some are self-explanatory. I can add a comment for `disableMergeWorker` though, as it may not be clear for someone who doesn't know merge workers.